tests/detect-ldap-result/test.yaml

```diff
@@ -0,0 +1,18 @@
+requires:
+  min-version: 8
+
+args:
+  - -k none
+  - --set stream.midstream=true
+
+pcap: ../ldap-unsolicited/ldap.pcap
```

Comment:
perhaps because it's using a different pcap file?

Reply:
This pcap ``../ldap-unsolicited/ldap.pcap`` is the one that I want to test. I was using the pcap from ``ldap-search`` and it was working, but I changed the pcap because I also want to test the ldap error message that is present here